diff -up googletest-release-1.8.1/googlemock/CMakeLists.txt.1~ googletest-release-1.8.1/googlemock/CMakeLists.txt
--- googletest-release-1.8.1/googlemock/CMakeLists.txt.1~	2018-12-11 15:34:21.539613211 +0100
+++ googletest-release-1.8.1/googlemock/CMakeLists.txt	2018-12-11 15:35:06.332005653 +0100
@@ -112,6 +112,8 @@ else()
   cxx_library(gmock_main "${cxx_strict}" src/gmock_main.cc)
   target_link_libraries(gmock_main PUBLIC gmock)
 endif()
+set_target_properties(gmock PROPERTIES VERSION 0.0 SOVERSION 0)
+set_target_properties(gmock_main PROPERTIES VERSION 0.0 SOVERSION 0)
 # If the CMake version supports it, attach header directory information
 # to the targets for when we are part of a parent build (ie being pulled
 # in via add_subdirectory() rather than being a standalone build).
diff -up googletest-release-1.8.1/googletest/CMakeLists.txt.1~ googletest-release-1.8.1/googletest/CMakeLists.txt
--- googletest-release-1.8.1/googletest/CMakeLists.txt.1~	2018-12-11 15:34:21.540613220 +0100
+++ googletest-release-1.8.1/googletest/CMakeLists.txt	2018-12-11 15:35:58.667478462 +0100
@@ -132,6 +132,8 @@ endif()
 # aggressive about warnings.
 cxx_library(gtest "${cxx_strict}" src/gtest-all.cc)
 cxx_library(gtest_main "${cxx_strict}" src/gtest_main.cc)
+set_target_properties(gtest PROPERTIES VERSION 0.0 SOVERSION 0)
+set_target_properties(gtest_main PROPERTIES VERSION 0.0 SOVERSION 0)
 # If the CMake version supports it, attach header directory information
 # to the targets for when we are part of a parent build (ie being pulled
 # in via add_subdirectory() rather than being a standalone build).
